TauAnalysis/MCEmbeddingTools/python/CleaningProcedure_cff.py

```diff
@@ -0,0 +1,134 @@
+import FWCore.ParameterSet.Config as cms
+
+MuonImput = cms.InputTag("selectedMuonsForEmbedding","","")
+
+cleanedsiPixelDigis = cms.EDProducer('PixelDigiCleaner',
+    MuonCollection = MuonImput,
+    oldCollection = cms.InputTag("siPixelDigis","","")
+)
+
+
+
+cleanedsiPixelClusters = cms.EDProducer('PixelCleaner',
```

Comment:
there are no references to this file in this PR.
Also, cleaner producer name is apparently PixelColCleaner.
Should this file be here?